schema/artist/index.js

```diff
@@ -377,113 +377,15 @@ const ArtistType = new GraphQLObjectType({
-            }),
-            // Highest tier group institutional shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: true,
-              is_reference: true,
-              highest_tier: true,
-              solo_show: false,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Highest tier group gallery shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: false,
-              is_reference: true,
-              highest_tier: true,
-              solo_show: false,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Lower tier solo institutional shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: true,
-              is_reference: true,
-              highest_tier: false,
-              solo_show: true,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Lower tier solo gallery shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: false,
-              is_reference: true,
-              highest_tier: false,
-              solo_show: true,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Lower tier group institutional shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: true,
-              is_reference: true,
-              highest_tier: false,
-              solo_show: false,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Lower tier group gallery shows
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              is_institution: false,
-              highest_tier: false,
-              is_reference: true,
-              solo_show: false,
-              at_a_fair: false,
-              visible_to_public: false,
-              size: options.size,
-            }),
-            // Fair booths
-            gravity('related/shows', {
-              artist_id: id,
-              sort: '-end_at',
-              at_a_fair: true,
-              visible_to_public: false,
-              size: options.size,
-            }),
-          ]).then(allShows => take(concat(...allShows), options.size));
+          return gravity('related/shows', {
+            artist_id: id,
+            sort: '-relevance,-start_at',
```

Comment:
I like this better than `-end_at`, observe this show: https://www.artsy.net/show/san-francisco-museum-of-modern-art-sfmoma-the-doris-and-donald-fisher-collection

It runs for 4 years?! Accurate or not, I feel like that's gaming the system a little bit- it's always going to be the first entry!